CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+### Release 2.2.1 (June 28, 2019)
+[Milestone#32](https://github.com/awslabs/amazon-kinesis-client/milestone/32)
+* Add periodic logging of the the state of the executor service.
```

Comment:
Can we be specific about which executor service?

Reply:
Updated to thread pool executor service